service/api/src/test/java/org/eclipse/kapua/service/config/ServiceXmlConfigPropertiesAdapterTest.java

```diff
@@ -0,0 +1,130 @@
+ *******************************************************************************/
+package org.eclipse.kapua.service.config;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.hamcrest.core.IsInstanceOf;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Category(JUnitTests.class)
+public class ServiceXmlConfigPropertiesAdapterTest extends Assert {
+
+    ServiceXmlConfigPropertiesAdapter serviceXmlConfigPropertiesAdapter;
+    Map[] expectedProperties;
+
+    @Before
+    public void initialize() {
+        serviceXmlConfigPropertiesAdapter = new ServiceXmlConfigPropertiesAdapter();
+
+        Map<String, Object> expectedProperty1 = new HashMap<>();
+        Map<String, Object> expectedProperty2 = new HashMap<>();
+        Map<String, Object> expectedProperty3 = new HashMap<>();
+        Map<String, Object> expectedProperty4 = new HashMap<>();
+        Map<String, Object> expectedProperty5 = new HashMap<>();
+        Map<String, Object> expectedProperty6 = new HashMap<>();
+        Map<String, Object> expectedProperty7 = new HashMap<>();
+        Map<String, Object> expectedProperty8 = new HashMap<>();
+        Map<String, Object> expectedProperty9 = new HashMap<>();
+        Map<String, Object> expectedProperty10 = new HashMap<>();
+        expectedProperty2.put(null, "47");
+        expectedProperty3.put(null, (long) 47);
+        expectedProperty4.put(null, 47d);
+        expectedProperty5.put(null, 47f);
+        expectedProperty6.put(null, 47);
+        expectedProperty7.put(null, (byte) 47);
+        expectedProperty8.put(null, '4');
+        expectedProperty9.put(null, false);
+        expectedProperty10.put(null, (short) 47);
+        expectedProperties = new Map[]{expectedProperty1, expectedProperty2, expectedProperty3, expectedProperty4, expectedProperty5, expectedProperty6, expectedProperty7, expectedProperty8, expectedProperty9, expectedProperty10};
+    }
+
+    @Test
+    public void marshalTest() {
+        Map<String, Object> props = new HashMap<>();
+        String stringValue = "String Value";
+        long longValue = 10L;
+        double doubleValue = 10.10d;
+        float floatValue = 10f;
+        int integerValue = 20;
+        byte byteValue = 10;
+        char characterValue = 'c';
+        boolean booleanValue = true;
+        short shortValue = 32767;
+        String[] stringArray = {null, "String Value 1", "String Value 2", "String Value 3"};
+        Long[] longArray = {10L, 20L, 20200L};
+        Double[] doubleArray = {10.123d, 1000.1234556d, 10.100000000000000000000d};
+        Float[] floatArray = {10.100000000000000000000f, 1010101010.10f, 1010101010.1f};
+        Integer[] integerArray = {-2000, -20, 0, 10, 20,};
+        Byte[] byteArray = {-128, -10, 0, 10, 127};
+        Character[] characterArray = {'c', 'h', 'a', 'r', 'a', 'c', 't', 'e', 'r'};
```

Comment:
Please add "null" as parameter, otherwise you will not cover lin 140 in the tested class.